examples/pypi_repository/BUILD

```diff
@@ -0,0 +1,32 @@
+
+pypi_repository(
+    name="pypi",
+    repository="pypi",
+)
+
+pypi_repository(
+    name="testpypi",
+    repository="testpypi",
+)
+
+pypi_repository(
+    name="privatepypi",
+    repository_url="https://foo:bar@example.com/",
+    username_env="PRIVATE_TWINE_USERNAME",
+    password_env="PRIVATE_TWINE_PASSWORD",
```

Comment:
I'm all for not having sensitive information in the BUILD file. However, I find it a bit awkward to tack the means of distribution of the information onto the name of the field (i.e. the `_env` suffix). How about simply calling it just `username`, and then supporting env expansion, so you could do `username="$PRIVATE_TWINE_USERNAME"`. That opens up for greater flexibility down the line too.. also, some may be ok with a hard coded username in the BUILD file, other don't. This way that will not force their hand either way.

Reply:
This is an excellent idea, will do :+1: